test/builtin/files_io/test_files.py

```diff
@@ -349,6 +348,52 @@ def test_private_doctests_files(str_expr, msgs, str_expected, fail_msg):
     )
 
 
+def test_open_read():
+    """ """
+    new_temp_file = NamedTemporaryFile(mode="r", delete=True)
+    name = canonic_filename(new_temp_file.name)
+    os.unlink(name)
+    check_evaluation(
+        str_expr=f'OpenRead["{name}"]',
+        str_expected=f"OpenRead[{name}]",
+        to_string_expr=True,
+        hold_expected=True,
+        failure_message=None,
+        expected_messages=(f"Cannot open {name}.",),
+    )
+
+
+# rocky: I don't understand what these are supposed to test.
+# (
```

Comment:
@mmatera if you can elucidate what is getting tested before, I am sure we can adapt this for pytest. 

I suspect the various complications that need to get addressed would be more easily, and robustly specified in Python rather than as a complex Mathics3 expression. And this time around, maybe we can be more expansive about what the intent is and what the test does.